src/nwb/NWBRecording.cpp

```diff
@@ -30,13 +30,15 @@ Status NWBRecording::openFile(const std::string& rootFolder,
                                            createIO(IOType, filename));
   nwbfile->initialize();
 
+  // create the datasets
+  nwbfile->createElectricalSeries(recordingArrays);
+
   // start the new recording
-  return nwbfile->createElectricalSeries(recordingArrays);
+  return nwbfile->startRecording();
```

Comment:
For now this is fine, but we should discuss in a separate issue how we want to structure the `openFile`. Currently this assumes that a user wants to setup an exephys recording with just ElectricalSeries. I think as we add more types, I think it would be easier to have the user call these manually, i.e,.a user would call:

1. `NWBRecording::openFile` (which would just initalize the file)
2. call `nwbrecording->nwbfile->createElectricalSeries` (and any other create methods needed)
3. call `nwbrecroding->startRecording`
```